### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by dhirajsb**
Add standard github templates for creating issues.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
+ - Device: [e.g. iPhone6]
+ - OS: [e.g. iOS8.1]
+ - Browser [e.g. stock browser, safari]
+ - Version [e.g. 22]
```
Comment (antisaling) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Are we worried about such frontends in this project? Also, we might specify more sensible software on the `Desktop` section.

File: .github/ISSUE_TEMPLATE/custom.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+name: Custom issue template
+about: Describe this issue template's purpose here.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+
```
Comment (antisaling) [inline .github/ISSUE_TEMPLATE/custom.md]: We can scrape that as it's also easy to create a non-fitting issue (I think there's no setting to disable that).

EDIT: for reference, look just below the boxes.

image

**Review by antisaling — COMMENTED** [review]
I also wonder whether it'd be useful to have standard prefixes (such as [BUG] or [FR] or auto-label these as "triage" or something).

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (dhirajsb) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Yeah, I took the standard templates for now. Doesn't mean we need all the bits. I'll remove this portion from the template for now. 
I'm sure we'll find things we don't use, or need to add as we create issues.

File: .github/ISSUE_TEMPLATE/custom.md (hunk 2 above)
Comment (dhirajsb) [reply .github/ISSUE_TEMPLATE/custom.md]: It's just a generic issue template for things such as chores, code formatting, etc. that might not fit in the bug/feature buckets.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (antisaling) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Sure thing, I like to keep it minimal so people don't get confused trying to interpret the significance of reporting their phone browser version or something unrelated.

File: .github/ISSUE_TEMPLATE/custom.md (hunk 2 above)
Comment (antisaling) [reply .github/ISSUE_TEMPLATE/custom.md]: Sure, I get it, it's just that there's a default option for that (I think those templates are mostly useful for external people trying to ask things tbh).

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (dhirajsb) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Fixed the template to remove front ends.

File: .github/ISSUE_TEMPLATE/custom.md (hunk 2 above)
Comment (dhirajsb) [reply .github/ISSUE_TEMPLATE/custom.md]: Changed this to a chores template.

**Comment by dhirajsb** [comment]
We should use standard prefixes and labels. Let's start with the list below:
* [bug]
* [feature]
* [chore]

**Review by tarilabs — APPROVED** [review]
added PR template as discussed previously on Slack; the other templates looks good to me, thank you 👍

**Review by lampajr — APPROVED** [review]

**Pull request merged**